plugins/woocommerce-blocks/assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -350,6 +350,13 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 			<ColorPanel colorTypes={ miniCartColorAttributes } />
 			<Noninteractive>
 				<button className="wc-block-mini-cart__button">
+					<QuantityBadge
+						count={ productCount }
+						iconColor={ iconColor }
+						productCountColor={ productCountColor }
+						icon={ miniCartIcon }
+						productCountVisibility={ productCountVisibility }
+					/>
```

Comment:
Just for curiosity: is there a particular reason for moving this component here?

Reply:
Yes it was the price of the mini cart, the requested change was to move the price from the left to the right side of the icon. I could do this with css only, but I wanted to avoid to use custom css as much as possible within the blocks.